src/data/icons/svg/TYC.svg

```diff
@@ -0,0 +1,13 @@
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#005B9F;}
+</style>
+<g id="LOGO_FINAL_2_">
```

Comment:
we can't afford to have `id=`, `class=` or `<style>` in these SVG because we will convert them back into react class.

Reply:
Okay thanks, we will update this PR soon.

Reply:
@gre updated the svg file :)